mdserver.js

```diff
@@ -0,0 +1,14 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const config = require('./lib/Config.js').default;
+const MetadataServer = require('arsenal').storage.metadata.server;
+
+if (config.backends.metadata === 'file') {
+    const mdServer = new MetadataServer(
+        { metadataPath: config.filePaths.metadataPath,
+          metadataPort: config.metadataDaemon.port,
+          log: config.log });
```

Comment:
This `config.log` is only the logging configuration, not a logger, unless that's the purpose.